.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 * @grafana/grafana-oncall-backend
 /grafana-plugin @grafana/grafana-oncall-frontend
-/docs @grafana/docs-gops @grafana/grafana-oncall
+/docs @grafana/grafana-oncall
```

Comment:
should we maybe add `@grafana/docs-squad` here in lieu of `@grafana/docs-gops`?

Reply:
We wouldn't want to use the @grafana/docs-squad here as our team workflow has these broke into smaller sub-teams. Michael and I did recently discussed updating this as well. But rather to update the @grafana/docs-gops to include a couple more writers and engineers.